awscli/customizations/globalargs.py

```diff
@@ -66,6 +58,23 @@ def _resolve_endpoint_url(value):
     return value
 
 
+def resolve_verify_ssl(parsed_args, session, **kwargs):
+    SYSTEM_CA_BUNDLE = '/etc/pki/tls/certs/ca-bundle.crt'
```

Comment:
I can do that, yes. When I wrote the code, I did some research on cross-platform support for determining the system cert bundle, but I got waaaaay out of my depth really quickly. So, yeah, this was a shortcut for my immediate needs.